monitoring/prometheus/ymls/prod.yml

```diff
@@ -46,9 +46,9 @@
     scheme: 'http'
     metrics_path: '/metrics'
     static_configs:
-      - targets: ['discoveryprovider2.audius.co:9091']
+      - targets: ['discoveryprovider4.audius.co:9091']
```

Comment:
the port is required here? shouldn't it be `discoveryprovider4.audius.co/prometheus_metrics`

Reply:
The port is required because this'll be the port for the push-gateway. I just realized though that it has to be an ip and not the domain name. Updating now

Reply:
ahh

Reply:
this is fine if it unblocks you for now, but we should not be exposing the port like this, instead we should assign it some dns record, should be really easy via cloudflare